theano/sandbox/rng_mrg.py

```diff
@@ -1292,8 +1292,17 @@ def uniform(self, size, low=0.0, high=1.0, ndim=None, dtype=None,
             The output data type. If dtype is not specified, it will be
             inferred from the dtype of low and high, but will be at
             least as precise as floatX.
-
+        nstreams:
+            The number of streams to use for sampling the random numbers
+            in parallel.
+        broadcastable:
```

Comment:
I understand the need to have control of the broadcastable flags, however I think it is better for the user to call `.patternbroadcast()` explicitly on the result, rather than adding `, broadcastable=` to the interface all operations that may need it.